sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -379,6 +464,9 @@ public String getId() {
     private <T> CosmosPagedFlux<T> queryItemsInternal(
         SqlQuerySpec sqlQuerySpec, FeedOptions feedOptions, Class<T> classType) {
         return UtilBridgeInternal.createCosmosPagedFlux(pagedFluxOptions -> {
+            String spanName = "queryItems." + this.getId() + "." + sqlQuerySpec.getQueryText();
```

Comment:
does this log the user query? is that ok? user query may have private info?
```java
"queryItems." + this.getId() + "." + sqlQuerySpec.getQueryText();
```

Reply:
done

Reply:
For jdbc, we only capture PreparedStatement query templates, and not the actual bind parameters provided by the user. We do something similar for MongoDB and other databases. is `querytext` a parameterized query template? or are bind parameters already embedded into it?

Reply:
Cosmos have two api for query , one take plain text with parameter embedded into it and second takes SqlQuerySpec , which is template based. For plain text query, code itself create SqlQuerySpec and use common code flow as above.
We have two option here - 
1. Only log query if user uses SqlQuerySpec  api , but this will be inconsistent.
2. Log for both as it is happening now , assuming for all the sensitive query user will use SqlQuerySpec  api.
3. Don't log query text any of the apis.

I would go with option 1 , as I can take inconsistency over possibility of leaking private info. Option 3 will be my second choice
Thoughts ?

Reply:
yes, #1 sounds good. we still want to capture a span for plain text queries, but use something more generic for the span name and `db.statement` attribute, e.g. the same value that you put into `db.instance` would be acceptable

Reply:
done